common/djangoapps/util/keyword_substitution.py

```diff
@@ -21,26 +21,19 @@
 """
 
 from django.contrib.auth.models import User
+from student.models import anonymous_id_for_user
 from xmodule.modulestore.django import modulestore
 
-KEYWORD_FUNCTION_MAP = {}
 
-
-def keyword_function_map_is_empty():
-    """
-    Checks if the keyword function map has been filled
-    """
-    return not bool(KEYWORD_FUNCTION_MAP)
-
-
-def add_keyword_function_map(mapping):
+def anonymous_id_from_user_id(user_id):
     """
-    Attaches the given keyword-function mapping to the existing one
+    Gets a user's anonymous id from their user id
     """
-    KEYWORD_FUNCTION_MAP.update(mapping)
+    user = User.objects.get(id=user_id)
```

Comment:
The code didn't before, and seeming as it's getting this `user_id` from the `User` table, I don't think we're at risk here